Loop/Managers/LoopSettingsManager.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  LoopSettingsManager.swift
+//  Loop
+//
+//  Created by Nathaniel Hamming on 2020-10-28.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LoopKit
+import LoopCore
+
+class LoopSettingsManager {
+    //TODO add tests
+    private let alertManager: AlertPresenter?
+
+    @Published var settings: LoopSettings
```

Comment:
Hm...not...quite what I was thinking, encapsulating `LoopSettings` in this way.  But I guess I see your approach.

Reply:
After a night to sleep on it, I'm going to take your suggestion. I realized I was overlooking using the protocol to also set. 

thanks for your thoughts.